tests/capture/test_capture_qnode.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
```suggestion
# Copyright 2024 Xanadu Quantum Technologies Inc.
```

Reply:
? I thought we did start to current?

Reply:
Oh, we do? I wasn't aware. `capture/__init__.py` and `capture/capture_meta.py`, for example, only have current in them, maybe we should fix those as well, then.

Reply:
Actually, `capture_qnode.py` also has "current" only. No need to iron this out in this PR, but we should take care of it somewhere, no?

Reply:
Brought up in main channel.  Thanks for starting this conversation.